cmd/run.go

```diff
@@ -497,7 +498,19 @@ func createSequenceSender(cfg config.Config, pool *pool.Pool, etmStorage *ethtxm
 }
 
 func runAggregator(ctx context.Context, c aggregator.Config, etherman *etherman.Client, ethTxManager *ethtxmanager.Client, st *state.State) {
-	agg, err := aggregator.New(c, st, ethTxManager, etherman)
+	var beethCli *agglayerClient.Client
+
+	if c.SettlementBackend == aggregator.AggLayer {
+		beethCli = agglayerClient.New(c.AggLayerURL)
+	}
+
+	// Load private key
+	pk, err := config.NewKeyFromKeystore(c.SequencerPrivateKey)
```

Comment:
only load pk if using aggLayer

Reply:
Fixed in https://github.com/0xPolygon/cdk-validium-node/pull/71/commits/e94685a03c767922cab92966633b43d0424a2186